### ISSUE TITLE
Border path doesn't match borderRadius for `Border (<position>)` mask

### ISSUE THREAD

**Issue by macintoshhelper** [airbnb/react-sketchapp#492]
| -------------------------------------------------------------------------------------------------
| Reporting a bug or issue
| -------------------------------------------------------------------------------------------------

**Expected behavior:**

![image](https://user-images.githubusercontent.com/6757532/79366303-da496180-7f43-11ea-920b-08fcb953feff.png)

**Observed behavior:**

![image](https://user-images.githubusercontent.com/6757532/79366037-65762780-7f43-11ea-9af5-f2263fc30d89.png)

**How to reproduce:**

```jsx
(
  <View
    style={{
      width: 60,
      height: 60,
      borderRadius: 6,
      borderTopLeftRadius: 12,
      borderBottomLeftRadius: 12,
      borderTopRightRadius: 0,
      borderBottomRightRadius: 0,
      borderRightWidth: 0,
      borderLeftWidth: 2,
      borderTopWidth: 2,
      borderBottomWidth: 2,
      borderColor: 'green',
      backgroundColor: 'blue',
    }}
  />
)
```

**Sketch version:**

`63.1`

**Please attach screenshots, a zip file of your project, and/or a link to your github project**

Included encapsulated code above, will link files in GitHub project shortly.

**Comment by mathieudutour** [comment]
oh interesting. That's actually pretty hard to fix

**Comment by macintoshhelper** [comment]
The border path masks could be replaced with a Svg path (to have instead of the `ShapeGroup` Rectangle). Would need a function to generate the `<Path>` data from the rounded corners.

**Comment by mathieudutour** [comment]
Yeah but when you have just a top border, the path is super weird

**Comment by SaiVamshiin** [comment]
You can try adjusting the values of the border-radius property to see if you can find a combination that works with the border path polyfill.
